doc/release_notes/release_notes_2.2.rst

```diff
@@ -0,0 +1,118 @@
+.. _release_notes_2.2:
```

Comment:
@dbkinder there is a number of ReST formatting issues in this doc. Will you take care of them in a final step or do we fix them now?